spond/spond.py

```diff
@@ -43,6 +44,22 @@ async def login(self):
         self.chat_url = result["url"]
         self.auth = result["auth"]
 
+    async def get_profile(self):
```

Comment:
I think the function name `get_profile()` should be `get_current_profile()` or similar, as the former sounds like a general purpose function to read an arbitrary item, like `get_event()`, `get_group()`, etc.